src/blocks/net.rs

```diff
@@ -709,159 +668,8 @@ struct IpAddrInfo {
-///     - `wpa_cli` failed to produce a valid UTF-8 SSID
-/// Returns Ok(None) if `wpa_cli` failed to produce a SSID.
-fn get_wpa_ssid(dev: &NetworkDevice) -> Result<Option<String>> {
-    let raw = exec_ssid_cmd("wpa_cli", &["status", "-i", &dev.device])?;
-
-    if raw.is_none() {
-        return Ok(None);
-    }
-
-    let raw = raw.unwrap();
-    let result = raw
-        .stdout
-        .split(|c| *c == b'\n')
-        .filter_map(|x| WPA_SSID_REGEX.captures_iter(x).next())
-        .filter_map(|x| x.get(1))
-        .next();
-
-    maybe_ssid_convert(result.map(|x| x.as_bytes()))
-}
-
-#[inline]
-/// Attempt to get the SSID the given device is connected to from nmcli.
-/// Returns Err if:
-///     - `nmcli` is not a valid command
-///     - failed to spawn a `nmcli` command
-///     - `nmcli` failed to produce a valid UTF-8 SSID
-/// Returns Ok(None) if `nmcli` failed to produce a SSID.
-fn get_nmcli_ssid(dev: &NetworkDevice) -> Result<Option<String>> {
-    let raw = exec_ssid_cmd(
-        "nmcli",
-        &["-g", "general.connection", "device", "show", &dev.device],
-    )?;
-
-    if raw.is_none() {
-        return Ok(None);
-    }
-
-    let raw = raw.unwrap();
-    let result = raw.stdout.split(|c| *c == b'\n').next();
-
-    maybe_ssid_convert(result)
-}
-
-#[inline]
-/// Attempt to get the SSID the given device is connected to from iwctl.
-/// Returns Err if
-///     - `iwctl` is not a valid command
-///     - failed to spawn a `iwctl` command
-///     - `iwctl` failed to produce a valid UTF-8 SSID
-/// Returns Ok(None) if `iwctl` failed to produce a SSID.
-fn get_iwctl_ssid(dev: &NetworkDevice) -> Result<Option<String>> {
-    let raw = exec_ssid_cmd("iwctl", &["station", &dev.device, "show"])?;
-
-    if raw.is_none() {
-        return Ok(None);
-    }
-
-    let raw = raw.unwrap();
-    let result = raw
-        .stdout
-        .split(|c| *c == b'\n')
-        .filter_map(|x| IWCTL_SSID_REGEX.captures_iter(x).next())
-        .filter_map(|x| x.get(1))
-        .next();
-
-    maybe_ssid_convert(result.map(|x| x.as_bytes()))
-}
-
-#[inline]
-fn exec_ssid_cmd<S, I, L>(cmd: S, args: I) -> Result<Option<std::process::Output>>
-where
-    S: AsRef<std::ffi::OsStr>,
-    I: IntoIterator<Item = L>,
-    L: AsRef<OsStr>,
-{
-    let raw = Command::new(&cmd).args(args).output();
-
-    if let Err(ref err) = raw {
-        if err.kind() == ErrorKind::NotFound {
-            return Ok(None);
-        }
-    }
-
-    raw.map(Some).block_error(
-        "net",
-        &format!(
-            "Failed to execute SSID query using {}",
-            cmd.as_ref().to_string_lossy()
-        ),
-    )
-}
-
-#[inline]
-fn maybe_ssid_convert(raw: Option<&[u8]>) -> Result<Option<String>> {
-    if let Some(raw_ssid) = raw {
-        String::from_utf8(decode_escaped_unicode(raw_ssid))
-            .block_error("net", "Non-UTF8 SSID")
-            .map(Some)
-    } else {
-        Ok(None)
-    }
-}
-
+// TODO use it?
```

Comment:
If not used then will it be a regression of the issue https://github.com/greshake/i3status-rust/pull/995 solved?